### PR TITLE
Documation01

### PR DESCRIPTION
**Pull request by minecraftchest1**
## Purpose

No associated Jira issue.

## Proposed changes

- Add doxygen documentation to ntoskrnl/ke/bug.c 

## TODO

- [ ] Document all functions

### CODE REVIEW

File: .github/workflows/Build.yml [hunk 1]
```diff
@@ -0,0 +1,179 @@
+name: Build
```
Comment (HBelusca) [inline .github/workflows/Build.yml:1]: What is this file for?

File: ntoskrnl/ke/bug.c [hunk 2]
```diff
@@ -626,7 +626,7 @@ KiDisplayBlueScreen(IN ULONG MessageId,
         InbvResetDisplay();
 
         /* Display blue screen */
-        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_BLUE);
+        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_GREEN);
```
Comment (HBelusca) [inline ntoskrnl/ke/bug.c:629]: That's not doc.

File: .github/workflows/Build.yml (hunk 1 above)
Comment (minecraftchest1) [reply .github/workflows/Build.yml:1]: Me setting up workflows. 
I'll see if I can remove and untrack that file.

File: ntoskrnl/ke/bug.c (hunk 2 above)
Comment (minecraftchest1) [reply ntoskrnl/ke/bug.c:629]: :(
I am going to start pulling from reactos/reactos master rather than minecraftchest1/reactos/master

**Comment by binarymaster** [comment]
Superseded by #3376.

**Pull request closed without merging**